frontend/javascripts/test/shaders/shader_syntax.spec.ts

```diff
@@ -48,9 +64,33 @@ test("Shader syntax: Ortho Mode + Segmentation - Mapping", (t: ExecutionContext<
     globalLayerCount: 2,
     colorLayerNames: ["color_layer_1", "color_layer_2"],
     textureLayerInfos: {
-      ["color_layer_1"]: { packingDegree: 4.0, dataTextureCount: 1 },
-      ["color_layer_2"]: { packingDegree: 4.0, dataTextureCount: 1 },
-      ["segmentationLayer"]: { packingDegree: 1.0, dataTextureCount: 4 },
+      ["color_layer_1"]: {
+        isColor: true,
+        packingDegree: 4.0,
+        dataTextureCount: 1,
+        isSigned: false,
+        glslPrefix: "",
+        unsanitizedName: "color_layer_1",
```

Comment:
`unsanitizedName` name is everywhere the same in this file and does not match the key of this dict / the actual layer name. Although as it currently does not yield any error I might in the future and therefore I would prefer to have it correct here :pray: